src/MSALWrapper/TokenResult.cs

```diff
@@ -70,6 +71,11 @@ public JsonWebToken JWT
         /// </summary>
         public bool IsSilent { get; internal set; }
 
+        /// <summary>
+        /// Gets the user's security identifier.
+        /// </summary>
+        public string Sid { get; internal set; }

```

Comment:
I might consider keeping this name as `SID` (all caps) because it's an acronyum not a word. I do not feel strongly about this though.